README.md

```diff
@@ -20,7 +20,16 @@ export interface TokenServiceClientOptions {
 
 The `jwt` option is required and the serviceUrl and env options are optional.  If the `env` option is not supplied it is determined via the window location.  If the `serviceUrl` option is not supplied it falls back to using the service url defined for the `env` and if that is not found defaults to the production service.  The serviceUrl option can be overridden via the `token-service-url` query parameter.
 
-## API
+## Example App (`example-app` subdirectory)
+
+http://token-service.concord.org/branch/travis-test/example-app/index.html
```

Comment:
Should this be changed to http://token-service.concord.org/example-app/index.html so it works after the merge?

Reply:
It should, thanks. Fixed in the last comment.